src/components/BlogResources.tsx

```diff
@@ -0,0 +1,56 @@
+import { FunctionComponent } from 'react'
+
+import { buttonStyle, buttonLocation } from '@data'
+
+import { ContentSection } from './ContentSection'
+
+interface BlogResources {
+    title?: string
+    posts: {
+        title: string
+        description: string
+        type: string
+        href: string
+        img?: {
+            src: string
+            alt: string
+        }
+    }[]
+}
+
+export const BlogResources: FunctionComponent<BlogResources> = ({ posts, title = 'Related Resources' }) => (
```

Comment:
Awesome! Small note, we may want to rename later `ResourceList` or something more generic semantically, because we often list non-blog items like guides, webinars or podcasts. The last component had this naming thing too

Reply:
That's a great idea! I like that! Let's go with `ResourceList`

Reply:
Fixed this in [0e8ade7](https://github.com/sourcegraph/about/pull/5654/commits/0e8ade722cb5c140f0b379d998446a4d08c687b9)